crates/etop-tui/src/app.rs

```diff
@@ -120,6 +127,7 @@ impl App {
                     //
                     // // etop setup
                     Action::BeginBlockSubscription => {
+                        let dataa = self.poll_rate.clone();
```

Comment:
can use better name like `poll_rate`

Reply:
> can use better name like `poll_rate`

ty for this review, changed thhe var name